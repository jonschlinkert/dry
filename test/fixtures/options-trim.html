{% extends "multiple" %}
{% block "header" %}
header: "block-multiple.html"
{% endblock %}
{% block "body" %}
body: "block-multiple.html"
{% endblock %}
{% block "footer" %}
footer: "block-multiple.html"
{% endblock %}