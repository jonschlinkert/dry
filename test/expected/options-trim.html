
head: "blocks/multiple.html"
header: "block-multiple.html"
body: "block-multiple.html"
footer: "block-multiple.html"
foot: "blocks/multiple.html"
